include/rmm/mr/cnmem_memory_resource.hpp

```diff
@@ -133,6 +133,27 @@ class cnmem_memory_resource final : public device_memory_resource {
     }
   }
 
+  /**---------------------------------------------------------------------------*
+   * @brief Get free and available memory for memory resource
+   *
+   * @throws std::runtime_error if we could not get cnmem free / total memory
+   *
+   * @param freeSize amount of currently free memory
+   * @param totalSize total memory available to resource
+   * @param stream the stream being executed on
+   *---------------------------------------------------------------------------**/
+  std::pair<size_t,size_t> do_get_mem_info( cudaStream_t stream){
+    size_t freeSize,totalSize;
```

Comment:
One declaration per line. Use `std::size_t`. snake_case. 
```suggestion
    std::size_t free_size;
    std::size_t total_size;
```

Reply:
Make this change on the other new functions as well.